google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataSettings.java

```diff
@@ -54,6 +54,10 @@
  * BigtableDataSettings settings = builder.build();
  * }</pre>
  *
+ * <p>{@link com.google.api.gax.rpc.TransportChannelProvider} provides thread-safe {@link
+ * io.grpc.ManagedChannel channels} for this service. Each of these channel manage connection
+ * pooling; means it automatically creates and destroys connection as they needed.
+ *
```

Comment:
I think this can should be a bullet point on line 39:
`The transport provider is configured with a channel pool that contains twice as many connections as CPUs`

Reply:
Thanks, Have updated with the same.